explorer/src/pages/AccountDetailsPage.tsx

```diff
@@ -108,17 +112,19 @@ const TOKEN_TABS_HIDDEN = [
 type Props = { address: string; tab?: string };
 export function AccountDetailsPage({ address, tab }: Props) {
   const fetchAccount = useFetchAccountInfo();
+  const fetchMetadata = useFetchMetadata();
```

Comment:
I'm not thrilled this is getting called for ALL account page loads. Going to try and gate this, or bail early. Generating the PDA is expensive and wasteful if we don't actually need it.

Reply:
Ended up dropping the Provider.